git/git-cheatsheet-de.md

```diff
@@ -103,17 +103,37 @@ Tag remote von origin löschen: `git push origin :<tagname>`
 
 ## Open-Source-Arbeit mit Forks
 
-Das Original eines Forks als `upstream`-Remote lokal hinzufügen:
+### Mit dem Fork verbinden
+Das Original eines Forks lokal clonen: `git clone <fork-repository> <zielverzeichnis>`
```

Comment:
Nein, das Original wollt ihr normalerweise nicht lokal haben, sondern direkt euren Fork.

Vielleicht wären noch ein paar Worte dazu gut, dass ihr vorher das Repository auf GitHub forken solltet.

Reply:
Das Zielverzeichnis ist optional - also entweder komplett weglassen oder sonst in Klammern schreiben, damit sichtbar ist, dass es optional ist.

Reply:
Beschreibung gändert